README.md

```diff
@@ -70,7 +70,7 @@ EOD
 ```bash
 mkdir app/${appName}/userparameters
 
-cat > app/${appName}/userparameters/${appName}.conf <<EOD
+cat > app/${appName}/userparameters/rabe.${appName}.conf <<EOD
```

Comment:
Can we have `rabe-${appName}.conf` here? The dot looks very counterintuitive to me.